tests/tacacs/test_ro_user.py

```diff
@@ -74,18 +78,24 @@ def test_ro_user_allowed_command(localhost, duthosts, rand_one_dut_hostname, cre
             '"sudo vtysh -c \'show ip bgp su\'"',
             '"sudo vtysh -n 0 -c \'show ip bgp su\'"',
             'sudo decode-syseeprom',
-            'sudo generate_dump',
+            'sudo generate_dump -s "5 secs ago"',
             'sudo lldpshow',
             'sudo pcieutil check',
             # 'sudo psuutil *',
             # 'sudo sfputil show *',
-            'sudo ip netns identify 1',
+            'sudo ip netns identify 1'
```

Comment:
I think keep the last comma is a good thing, I saw changes added new line but missed adding comma to the previous line.